packages/wrangler/src/config/config.ts

```diff
@@ -184,7 +184,7 @@ export interface DevConfig {
   /**
    * Host to forward requests to, defaults to the host of the first route of project
    */
-  host: Config["route"];
+  host: string | undefined;
```

Comment:
I think typing this as `Config["route"]` was a mistake, since in our validation we check that it is a `string`.